hubmap/dataportal/views.py

```diff
@@ -190,3 +220,91 @@ def serialize_multi_dim_counts(data: xr.DataArray):
         pass
 
     return list_for_frontend
+
+
+def globus(request):
+    uuid = None
+    access_token = None
+    refresh_token = None
+    if not hasattr(request, 'user'):
+        request.user = AnonymousUser
+
+    elif request.user.is_authenticated:
+        response = HttpResponseRedirect('http://localhost:3000/')
```

Comment:
This is one of the `localhost` references you mentioned on the call?

Reply:
sort of, but i have some local changes, also refer: https://github.com/hubmapconsortium/hubmap-data-portal/commit/4fde31c08a5020b32b52b60daa2571c140ca4e84